cc3d/run_script.py

```diff
@@ -89,7 +89,10 @@ def handle_error():
 
     rollbackImporter = RollbackImporter()
 
-    persistent_globals.simulation_file_name = cc3d_sim_fname
+    current_dir = args.current_dir if args.current_dir else ''
+    cc3d_sim_fname_abs = join(current_dir, cc3d_sim_fname)
```

Comment:
at some point we need to switch to pathlib.Path but this is a task for when we are bored :)